core/platform.js

```diff
@@ -75,7 +75,7 @@ Platform.prototype.messageRxd = function(api, event) {
                 this.handleTransaction(this.loadedModules[i], runArgs);
             }
             catch (e) {
-                api.sendMessage(event.body + ' fucked up. Damn you ' + event.sender_name + ".", event.thread_id);
+                api.sendMessage(event.body + ' is boken. Damn you ' + event.sender_name + '.', event.thread_id);
```

Comment:
Technically it might not be broken, just throwing an error at you. Also shouldn't "damn you" also be changed?